docs/learning/quantity/index.md

```diff
@@ -46,6 +50,34 @@ If you're developing a frontend application that takes advantage of the core qua
 
 For a detailed description of all the setting supported by FormatProp see the EC documentation on [Format](../../bis/ec/ec-format.md).
```

Comment:
I think we should mention the new fields on that page, too ( leads to here https://www.itwinjs.org/bis/ec/ec-format/ )